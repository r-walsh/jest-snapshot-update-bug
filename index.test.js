const { add } = require('./');

describe('add', () => {
  it('adds two numbers', () => {
    expect(add(1, 2)).toMatchSnapshot();
  });
});

