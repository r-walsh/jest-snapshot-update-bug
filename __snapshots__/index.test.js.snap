// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add adds two numbers 1`] = `3`;
